### ISSUE TITLE
Regression in 4.4.0 : build node10-win64 from a linux host produces an empty executable

### ISSUE THREAD

**Issue by AxelTerizaki** [zeit/pkg#686]
Hello :)

I just noticed my dev-branch builds weren't working anymore on Windows.

The executable is built on a Linux host (inside a docker CI/CD runner) and are offered to those who want to test out dev buidls of our app.

One user told me that the exe file wasn't doing anything anymore. When he tries to open it, nothing happens. When he tries to run it from a cmd.exe command prompt, the program displays an empty line and returns to prompt as if it did absolutely nothing.

I'm attaching the log here : 

[pkg.log](https://github.com/zeit/pkg/files/3188245/pkg.log)

Note that there's some failure on safefs but it doesn't usually keep the executable from working properly. 

I just rebuilt a version of my software with pkg 4.3.8 and it works fine.

Also I should mention that building an executables on 4.4.0 under Windows for Windows works fine. It's building under Linux for Windows that seems to produce a weird executable.

If I can help in any way to diagnose the issue, please say so.

**Comment by boredland** [comment]
I can reproduce this with this public project: https://github.com/APIDevTools/swagger-cli

```
[jonass@jonas-vm ~]$ npm -v
6.9.0
[jonass@jonas-vm ~]$ node -v
v11.15.0
```
Command run: `pkg --target node10-linux-x64,node10-win-x64 . --out-path ../target/bin/`

Result: working linux binary, but when I run the windows binary on w10 nothing happens.

**Comment by AxelTerizaki** [comment]
Apparently this is fixed with pkg 4.4.2 now. Why, I have no idea, but thanks for the fix :)
